packages/models-library/src/models_library/projects_access.py

```diff
@@ -26,33 +25,22 @@ class AccessRights(BaseModel):
     read: bool = Field(..., description="has read access")
     write: bool = Field(..., description="has write access")
     delete: bool = Field(..., description="has deletion rights")
-
-    class Config:
-        extra = Extra.forbid
-
-
-class PositiveIntWithExclusiveMinimumRemoved(PositiveInt):
-    # As we are trying to match this Pydantic model to a historical json schema "project-v0.0.1" we need to remove this
-    # Pydantic does not support exclusiveMinimum boolean https://github.com/pydantic/pydantic/issues/4108
-    @classmethod
-    def __modify_schema__(cls, field_schema):
-        field_schema.pop("exclusiveMinimum", None)
+    model_config = ConfigDict(extra="forbid")
 
 
 class Owner(BaseModel):
-    user_id: PositiveIntWithExclusiveMinimumRemoved = Field(
-        ..., description="Owner's user id"
-    )
+    user_id: PositiveInt = Field(..., description="Owner's user id")
```

Comment:
not necessary anymore?

Reply:
No `exclusiveMinimum` in the new definition, so nothing to remove.